src/main/resources/data/minelabs/loot_tables/entities/balloon.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "minecraft:entity",
+  "pools": [
+    {
+      "rolls": 2,
+      "entries": [
+        {
+          "type": "minecraft:item",
+          "name": "minecraft:rabbit_hide",
```

Comment:
The crafting recipe uses normal Leather.
Either change this to normal Leather.
Or change the crafting recipe.

Consistency.

Reply:
I don't like that it can drop leather. It should drop something smaller and since 4 rabbit hide = 1 leather...
But I might change the recipe